js/index.js

```diff
@@ -0,0 +1,52 @@
+
+
+//creating a footer
```

Comment:
Comments are really helpful when they provide additional context - more so why a coding/design decision was made. Some of your comments could be removed to improve readability.